Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.yml

```
@@ -9,18 +9,32 @@ configuration:
   required: true
   type: 0
 - display: API Token
-  name: apitoken
+  name: credential_token
+  required: true
```

defined a new param as required will require already configured instances to be reconfigured 
```suggestion
  required: false
```